Source/WebCore/rendering/RenderThemeIOS.mm

```diff
@@ -721,14 +721,26 @@ static bool renderThemePaintSwitchTrack(OptionSet<ControlStyle::State>, const Re
 }
 #endif
 
+// FIXME: Except for outline and disabled styling, we should deduplicate this with the generic code.
```

Comment:
Seems like we should at least do that in this PR for the `switch (style.display())` duplication? I don't imagine it will be much extra work.